docs/standard/serialization/snippets/system-text-json-how-to/vb/ValueTextEqualsExample.vb

```diff
@@ -0,0 +1,6 @@
+﻿' <DefineUtf8Var>
+' https://docs.microsoft.com/dotnet/standard/serialization/system-text-json-how-to#visual-basic-support
+' </DefineUtf8Var>
+' <UseUtf8Var>
+' https://docs.microsoft.com/dotnet/standard/serialization/system-text-json-how-to#visual-basic-support
+' </UseUtf8Var>
```

Comment:
What does this code mean? Is this vb file rendered in any way?

Reply:
Same as the previous -- it's a comment for snippets that are n/a in VB, I added an explanation comment preceding the URL in all cases of this comment.